CMakeLists.txt

```diff
@@ -645,7 +645,7 @@ if(QML_DEBUGGING)
 endif()
 
 
-if(NOT MSVC)
+if(NOT MSVC AND NOT HAIKU)
```

Comment:
Hm, -Werror is only used in debug builds, which should not affect release or release with debug builds. I would be interested in what warnings there are on Haiku, that aren't on other platforms? Maybe we can fix them instead?

Reply:
Well, I guess there is no reason to hold this up from getting merged, just to wait for a reply :D